docs/command_reference.md

```diff
@@ -0,0 +1,279 @@
+<!--[metadata]>
++++
+title = "Notary Command Reference"
+description = "Notary command reference"
+keywords = ["docker, notary, command, notary command, reference"]
+[menu.main]
+parent="mn_notary"
+weight=99
++++
+<![end-metadata]-->
+
+# Notary Command Reference
+
+## Terminology Reference
+1. **GUN**: Notary uses Globally Unique Names (GUNs) to identify trusted collections.
+2. **Target**: Notary refers to the files the framework is used to distribute as "target files".  Target files are opaque to the framework. Whether target files are packages containing multiple files, single text files, or executable binaries is irrelevant to Notary.
+3. **Trusted Collection**: A trusted collection is a set of target files of interest to the application.
+4. **Notary Repository**: A notary repository refers to the set of signed metadata files that describe a trusted collection.
+5. **Key roles**:
+    - **Root**: The root role delegates trust to specific keys trusted for all other top-level roles used in the system.
```

Comment:
I'd rather not link to specific parts of the TUF spec in too many places in the command reference, since I feel like that's more relevant to the other documentation that explains more of the TUF concepts.  Do you think just the general TUF spec link I have below is enough?